docs/.readthedocs.yaml

```diff
@@ -21,11 +31,10 @@ sphinx:
   fail_on_warning: true
 
 # If using Sphinx, optionally build your docs in additional formats such as PDF
-# PDF disabled because of https://github.com/canonical/real-time-ubuntu-docs/issues/84
-# formats:
-#   - pdf
+formats:
+- pdf
```

Comment:
Isn't this going to re-trigger the pdf generation issue?
There is no error, but I don't see the build generating the pdf either: https://app.readthedocs.com/projects/canonical-real-time-ubuntu-documentation/builds/2873089/